package-lock.json

```
@@ -75,7 +75,7 @@
         "coveralls": "3.0.2",
         "eslint": "8.46.0",
         "eslint-config-prettier": "8.1.0",
-        "eslint-plugin-import": "*",
+        "eslint-plugin-import": "latest",
```

Consider reverting this unrelated change.
```suggestion
        "eslint-plugin-import": "*",
```

---

Thank you, I reverted.😊